gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/small_tests/platinum_NA12878_hg38_10K_lines_from_file.json

```diff
@@ -0,0 +1,48 @@
+[
```

Comment:
It might be confusion, this test actually means the input file already contains the annotation information, and we would like to parse the annotation fields into different columns (by using `--annotation_fields` flag). `gnomad_genomes_GRCh37_chrX_head2500_run_vep` is one example that runs annotation pipelines.

Reply:
Ahh added one more test for gnomad. Left plat test as well, just in case.